make/modules/java.base/Lib.gmk

```diff
@@ -213,8 +213,10 @@ $(eval $(call SetupJdkLibrary, BUILD_SYSLOOKUPLIB, \
     CXXFLAGS := $(CXXFLAGS_JDKLIB), \
     LDFLAGS := $(LDFLAGS_JDKLIB), \
     LDFLAGS_linux := -Wl$(COMMA)--no-as-needed, \
+    LDFLAGS_aix := -brtl  -bloadmap:/home/hotspot/openjdk/symbol.log -bE:/home/hotspot/tmp/all-libs/1.exp   , \
```

Comment:
These files need to get added somewhere. Maybe symbols could get added to make/data/hotspot-symbols/symbols-aix? Or to new files in the same directory (with aix in the file names)?
Also, please remove extra whitespaces.